### PR TITLE
chore: migrate from CLA to DCO

### PR DESCRIPTION
**Pull request by droshev**
#### Please provide a link to the associated issue.
fixes: #3017

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -65,29 +65,14 @@ You are welcome to contribute code to Fundamental Library for Angular in order t
 
 There are two important things to consider:
 
-1.  You must be aware of the Apache License (which describes contributions) and **agree to the Contributors License Agreement**. This is common practice in all major Open Source projects. If you are an individual contributor, use _[CLA assistant](https://cla-assistant.io/)_. CLA assistant is an open source tool that integrates with GitHub and enables a one-click-experience for accepting the CLA. If you are contributing on behalf of a company, see the [Company Contributors](#company-contributors) section below for details.
+1.  You must be aware of the Apache License (which describes contributions) and **accept the Developer Certificate of Origin**. This is common practice in major Open Source projects. To make this process as simple as possible, we are using *[CLA assistant](https://cla-assistant.io/)* for individual contributions. CLA assistant is an open source tool that integrates with GitHub very well and enables a one-click experience for accepting the CLA. For company contributors, special rules apply. See the respective section below for details.
```
Comment (jessmetivier) [inline CONTRIBUTING.md:68]: 1. You must be familiar with the Apache License (that describes contributions) and **accept the Developer Certificate of Origin**. This is a common practice in major open source projects. To make this process as simple as possible, we are using *[CLA assistant](https://cla-assistant.io/)* for individual contributions. CLA assistant is an open source tool that integrates well with GitHub, providing users with an easy one-click experience when accepting the CLA. For company contributors, special rules apply. See the respective section below for details.

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+ SAP or third-party products or services developed outside of this project
+ (“External Products”).
+ “APIs” means application programming interfaces, as well as their respective
+ specifications and implementing code that allows software to communicate with
+ other software.
+ API Calls to External Products are not licensed under the open source license
+ that governs this project. The use of such API Calls and related External
+ Products are subject to applicable additional agreements with the relevant
+ provider of the External Products. In no event shall the open source license
+ that governs this project grant any rights in or to any External Products,or
```
Comment (jessmetivier) [inline .reuse/dep5:16]: Space comma after Products,

File: .reuse/dep5 [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+ alter, expand or supersede any terms of the applicable additional agreements.
+ If you have a valid license agreement with SAP for the use of a particular SAP
+ External Product, then you may make use of any API Calls included in this
+ project’s code for that SAP External Product, subject to the terms of such
+ license agreement. If you do not have a valid license agreement for the use of
+ a particular SAP External Product, then you may only make use of any API Calls
```
Comment (jessmetivier) [inline .reuse/dep5:22]: Rearranged sentence for better clarity: "Then you may only make use of this project's API Calls for internal, non-productive, and non-commercial test and evaluation of such API Calls for that SAP External Product."

File: .reuse/dep5 [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ in this project for that SAP External Product for your internal, non-productive
+ and non-commercial test and evaluation of such API Calls. Nothing herein grants
+ you any rights to use or access any SAP External Product, or provide any third
+ parties the right to use of access any SAP External Product, through API Calls.
```
Comment (jessmetivier) [inline .reuse/dep5:26]: use or access* 
SAP External Products*

File: .reuse/dep5 [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+ that governs this project grant any rights in or to any External Products,or
+ alter, expand or supersede any terms of the applicable additional agreements.
+ If you have a valid license agreement with SAP for the use of a particular SAP
+ External Product, then you may make use of any API Calls included in this
+ project’s code for that SAP External Product, subject to the terms of such
+ license agreement. If you do not have a valid license agreement for the use of
+ a particular SAP External Product, then you may only make use of any API Calls
+ in this project for that SAP External Product for your internal, non-productive
+ and non-commercial test and evaluation of such API Calls. Nothing herein grants
+ you any rights to use or access any SAP External Product, or provide any third
```
Comment (jessmetivier) [inline .reuse/dep5:25]: SAP External Products*

File: .reuse/dep5 (hunk 4 above)
Comment (jessmetivier) [reply .reuse/dep5:26]: no comma after Products

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -65,29 +65,14 @@ You are welcome to contribute code to Fundamental Library for Angular in order t
...
-the [Corporate Contributor License Agreement](/docs/SAP%20Corporate%20Contributor%20License%20Agreement.pdf) form. Then either:
-
--   Scan it and e-mail it to [opensource@sap.com](mailto:opensource@sap.com),
--   Fax it to: +49 6227 78-45813,
--   Send it by mail to: _Industry Standards & Open Source Team, Dietmar-Hopp-Allee 16, 69190 Walldorf, Germany_.
-
-The form lists all employees who are authorized to contribute on behalf of your company. When this list changes, please let us know.
+## Developer Certificate of Origin (DCO)
 
+Due to legal reasons, contributors will be asked to accept a DCO before they submit the first pull request to this projects, this happens in an automated fashion during the submission process. SAP uses [the standard DCO text of the Linux Foundation](https://developercertificate.org/).
```
Comment (jessmetivier) [inline CONTRIBUTING.md:75]: before they submit **their** first pull request*

Comment (jessmetivier) [reply CONTRIBUTING.md:75]: to this project* (singular)

Comment (jessmetivier) [reply CONTRIBUTING.md:75]: "Due to legal reasons, contributors will be asked to accept a DCO before they submit their first pull request to this project. This process will happen in an automated fashion during a submission."

**Review by jessmetivier — CHANGES_REQUESTED** [review]
Just a few kinks to irons out!

**Review by jessmetivier — APPROVED** [review]

**Review by Vanessa-Cusmich — APPROVED** [review]
LGTM

**Pull request merged**